src/seedu/addressbook/parser/Parser.java

```diff
@@ -102,6 +110,8 @@ public Command parseCommand(String userInput) {
         case HelpCommand.COMMAND_WORD: // Fallthrough
         default:
             return new HelpCommand();
+
+
```

Comment:
It's better to delete any empty lines, I think.

Reply:
It's ok to have empty lines to make the codes clearer. But do not leave any white spaces in the empty lines.